package.json

```diff
@@ -99,8 +99,8 @@
     "@types/lodash": "^4.17.0",
     "@types/lodash-es": "^4.17.12",
     "@types/mocha": "^10.0.7",
-    "@types/node": "^22.1.0",
-    "@types/oidc-provider": "^8.4.4",
+    "@types/node": "^22.5.0",
+    "@types/oidc-provider": "^8.5.2",
```

Comment:
might be usefull to keep this syncronised with the correponding implementation :

"oidc-provider": "^8.4.6"
"node": "20.12.2"

Reply:
A vérifié l'update à pour but de récupérer du type ;)
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/oidc-provider/index.d.ts#L2169-L2170